modules/ROOT/pages/install-validate.adoc

```diff
@@ -6,31 +6,26 @@ endif::[]
 
 To verify your installation and configuration:
 
-. If you are using JSON Web Tokens and not impersonation, verify the Named Credentials.
-.. Open *Setup*.
-.. Search for *Named Credentials*.
-.. Locate *anypoint.mulesoft* and click *Edit*.
-.. Verify that the list *External Data User Authentications* shows your administrative Anypoint user.
-. If you are using impersonation and not JSON Web Tokens, verify the Exchange External Data Source status is *Authenticated*.
-.. Open *Setup*.
-.. Search for *External Data Sources*.
-.. In the list select *Exchange*.
-.. Verify that the *Authentication Status* is *Authenticated*.
+. Open the Salesforce Application Launcher.
```

Comment:
It looks like this is really presented as "App Launcher," although the UI shows *App launcher*...of course....

https://www.lightningdesignsystem.com/components/app-launcher/

Reply:
Good point! Updated to "App".